@xen-orchestra/lite/src/components/AppNavigation.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <nav
+    v-if="isDesktop || isOpen"
+    ref="navElement"
+    :class="{ collapsible: isMobile }"
+    class="app-navigation"
+  >
+    <InfraPoolList />
+  </nav>
+</template>
+
+<script lang="ts" setup>
+import InfraPoolList from "@/components/infra/InfraPoolList.vue";
+import { useNavigationStore } from "@/stores/navigation.store";
+import { useUiStore } from "@/stores/ui.store";
+import { onClickOutside, whenever } from "@vueuse/core";
+import { storeToRefs } from "pinia";
+import { ref } from "vue";
+
+const uiStore = useUiStore();
+const { isMobile, isDesktop } = storeToRefs(uiStore);
+
+const navigationStore = useNavigationStore();
+const { isOpen, trigger } = storeToRefs(navigationStore);
+
+const navElement = ref();
+
+whenever(isOpen, () => {
+  const unregisterEvent = onClickOutside(
+    navElement,
+    () => {
+      isOpen.value = false;
+      unregisterEvent?.();
+    },
+    {
+      ignore: [trigger],
```

Comment:
Yes, exactly.

Since the `trigger` is outside the navigation button, first `isOpen.value` would be set to `false` by `clickOutside`, but immediately would be set back to `true` by the `trigger`